components/menus/QiskitOrgMenu.vue

```diff
@@ -72,16 +81,13 @@
 import Vue from 'vue'
 import { Component, Prop } from 'vue-property-decorator'
 
+import { QISKIT_URL, orderedQiskitElements, orderedCommunitySubLinks } from '~/constants/links'
+
 @Component
 export default class extends Vue {
-  @Prop({
-    type: Array,
-    default: () => [
-      { to: '/education', label: 'Education' },
-      { to: '/advocates', label: 'Advocates' },
-      { to: '/experiments', label: 'Experiments' }
-    ]
-  }) links
+  @Prop({ type: String, default: QISKIT_URL }) baseUrl
```

Comment:
`baseUrl` has a [well known meaning in the Web world](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base). What about `qiskitHomeUrl` or something else?